src/lib/lib.gni

```diff
@@ -20,4 +20,7 @@ declare_args() {
   # a Commissioner or Administrator that employs different or more complex
   # logic, this should be set to false.
   chip_build_default_attestation_verifier = true
+
+  # Set to true to enable device-specific attestation credentials
+  chip_build_device_attestation_credentials = false
```

Comment:
Suggest rename to `chip_build_platform_attestation_credentials_provider`

Reply:
how is this working for other platforms ?

Reply:
Renamed. This variable may be used by any platform that wants to implement actual attestation credentials, but since it is used only for EFR32, presently it won't have any effect on other platforms.